src/mcp_can.cpp

```diff
@@ -191,7 +191,13 @@ bool MCP_CAN::mcp2515_isFastSupported(pin_t _pin)
     {
         return false;
     }
-    Hal_Pin_Type type = HAL_Pin_Map()[_pin].type;
+    auto type =
+#if (SYSTEM_VERSION < SYSTEM_VERSION_v500ALPHA1)
```

Comment:
Will this compile for versions of DeviceOS where `SYSTEM_VERSION_v500ALPHA1` isn't defined?

Reply:
Good catch.  It will now.